build-libs/redirects.js

```diff
@@ -179,6 +190,9 @@ async function buildProductRedirects() {
 	// Fetch author-oriented redirects from product repos,
 	// and merge those with dev-oriented redirects from
 	// within this repository
+	if (process.env.SKIP_BUILD_PRODUCT_REDIRECTS) {
+		return []
+	}
```

Comment:
I added this so that our bundle analysis workflow doesn't attempt to fetch redirects from the GH API and as a result require a GitHub PAT.